charts/sysbox/templates/sysbox/sysbox-deploy-rbac.yaml

```diff
@@ -0,0 +1,40 @@
+---
+# CHANGES
```

Comment:
link to original manifests?

Reply:
good idea, linked in readme but files is a good idea